server/src/main/java/com/flipkart/varadhi/services/SubscriptionService.java

```diff
@@ -11,11 +15,12 @@
 import static com.flipkart.varadhi.entities.VersionedEntity.INITIAL_VERSION;
 
 public class SubscriptionService {
-
     private final MetaStore metaStore;
+    private final ControllerOpHandler controllerOpHandler;
 
-    public SubscriptionService(MetaStore metaStore) {
+    public SubscriptionService(MetaStore metaStore, ControllerOpHandler controllerOpHandler) {
```

Comment:
why is subscriptionService taking controllerOpHandler as dependency? What does it mean?

Reply:
this is renamed to controllerApi